README.md

```diff
@@ -62,4 +62,9 @@ const Api = app.plugin(RPC, {handlers, fetch, EventEmitter});
 const instance = Api.of(ctx)
 ```
 
-- `instance: Object` - Has the same method names and signatures as their counterparts in `handlers`.
+- `instance.request(method: string, args: any)` - make an rpc call to the `method` handler with `args`.
```

Comment:
The word `request` refers to an implementation detail. Maybe we should call it `call`, `invoke` or `dispatch`?

Reply:
I thought about `call` but didn't want to use it as I thought it could be confusing with `function.call`. Tbh, I don't think the naming matters to much here, since this library isn't designed to be used directly by users.